gradle.properties

```diff
@@ -17,8 +17,8 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-VERSION_CODE=9
-VERSION_NAME=1.3.0-SNAPSHOT
+VERSION_CODE=10
+VERSION_NAME=1.3.0
```

Comment:
idea plan. I am going to push another commit on top of this, and tag the in the middle commit. So that when we merge, it will be snapshot for the next release.